addons/kibana/6.8.x/kibana-8.yaml

```diff
@@ -82,11 +82,20 @@ spec:
                     -H "Content-Type: application/json; charset=utf-8" -H 'kbn-xsrf: true' \
                     -d '{"attributes":{"title":"'"${index}"'-*","timeFieldName":"@ts","fields":"[]"}}'
                 fi
+                # check or exit, to redo
+                if [[ "$(curl -s -o /dev/null -w '%{http_code}\n' -L ${KB_URL}/api/saved_objects/index-pattern/${index})" != "200" ]]; then
```

Comment:
I like the patter for checking you establish on line 92 and line 95, why not convert this also?

Reply:
I thought setting default at new setup and always after restart isn't a bad idea.
So we just check if the default set succeeded with checking HTTP response code after POST operation

Reply:
confused myself 🤦 
it is done as wished 😄